packages/hoprd/hopr-admin/src/utils/command.ts

```diff
@@ -135,21 +188,34 @@ export abstract class Command {
   }
 }
 
-export type CmdParameter = [type: CmdTypes, name: string, optional?: boolean]
-type CmdTypes =
+/**
+ * All possible command types
+ */
+export type CmdTypes =
   | 'hoprAddress'
   | 'nativeAddress'
   | 'hoprAddressOrAlias'
   | 'hoprOrNative'
+  | 'direction'
+  | 'constant'
   | 'number'
   | 'string'
+  | 'arbitrary'
   | 'boolean'
-  | 'constant'
-type CmdArg<I, O, R> = [description: string, validation: (v: I, ops: O) => [valid: boolean, value: R]]
+export type { ChannelDirection } from './api'
+export type HoprOrNative = 'hopr' | 'native'
```

Comment:
This is not used anymore, is it?

Reply:
fixed in 20f8fc7b72b19ee260d997c6b1453439ed607cc4